Doc/library/multiprocessing.shared_memory.rst

```diff
@@ -20,8 +20,8 @@ and management of shared memory to be accessed by one or more processes
 on a multicore or symmetric multiprocessor (SMP) machine.  To assist with
 the life-cycle management of shared memory especially across distinct
 processes, a :class:`~multiprocessing.managers.BaseManager` subclass,
-:class:`SharedMemoryManager`, is also provided in the
-``multiprocessing.managers`` module.
+:class:`multiprocessing.managers.SharedMemoryManager`, is also provided in the
```

Comment:
Squiggle it to display just `SharedMemoryManager`:
```suggestion
:class:`~multiprocessing.managers.SharedMemoryManager`, is also provided in the
```

Reply:
Sure; I'm fine with either. The original text surprised me a little bit, which is why I made it more explicit in the first place.